pmm-app/tests/pages/dashboardPage.js

```diff
@@ -370,19 +370,20 @@ module.exports = {
   },
 
   async expandRows(sectionsToExpand) {
-    const sections;
+    let sections;
     if (typeof sectionsToExpand === 'string') {
       sections = [sectionsToExpand];
     } else {
       sections = sectionsToExpand;
     }
     for (let i = 0; i < sections.length; i++) {
-      const sectionName = sections[i].toString().split('(');
-      const rowToExpand = `${this.fields.collapsedDashboardRow}[contains(text(), '${sectionName[0]}')]`;
+      let sectionName = sections[i].toString().split('(');
```

Comment:
@puneet0191 Is there a reason to drop `const` in favor of `let`?

Reply:
@atymchuk I think I keep getting this: `Missing initializer in const declaration` so I wanted to avoid it.

Reply:
That's because your IDE is not properly set up.

Reply:
I was getting it while running the tests